tekton/predicates_test.go

```diff
@@ -96,8 +95,23 @@ var _ = Describe("Predicates", func() {
 			Expect(instance.Update(contextEvent)).To(BeFalse())
 
 			newPipelineRun.Annotations["chains.tekton.dev/signed"] = "true"
-			Expect(instance.Update(contextEvent)).To(BeTrue())
-			contextEvent.ObjectNew = &tektonv1beta1.TaskRun{}
+			Expect(instance.Update(contextEvent)).To(BeFalse())
+		})
+
+		It("should return false when an updated event is received for a failed PipelineRun and no signed", func() {
+			// also failed pipelines are signed by tekton chains, test it
```

Comment:
```suggestion
		It("should return false when an updated event is received for a succeeded PipelineRun and no signed", func() {
```

Reply:
Ack, updated, thanks for the catch

Reply:
Should we also change `failed PipelineRun` to `succeeded PipelineRun`?

Reply:
Yes, totally missed it on the first pass, updated

Reply:
Thank you!